clang/lib/CodeGen/Targets/RISCV.cpp

```diff
@@ -251,6 +253,15 @@ bool RISCVABIInfo::detectFPCCEligibleStructHelper(QualType Ty, CharUnits CurOff,
         // bitwidth is XLen or less.
         if (getContext().getTypeSize(QTy) > XLen && BitWidth <= XLen)
           QTy = getContext().getIntTypeForBitwidth(XLen, false);
+        // Trim type to alignment/bitwidth if that is possible
+        else if (getContext().getTypeSize(QTy) > Alignment &&
+                 getContext().getTypeSize(QTy) > BitWidth) {
+          bool isSigned =
```

Comment:
Variable names should be capitalized per coding standards

Reply:
Capitalized in the latest push